servers/quarkus-server/src/main/java/org/projectnessie/server/config/QuarkusServerConfig.java

```diff
@@ -16,18 +16,21 @@
 package org.projectnessie.server.config;
 
 import io.quarkus.arc.config.ConfigProperties;
-import org.eclipse.microprofile.config.inject.ConfigProperty;
+import io.smallrye.config.WithDefault;
+import io.smallrye.config.WithName;
 import org.projectnessie.services.config.ServerConfig;
 
 /** Nessie server config for Quarkus. */
 @ConfigProperties(prefix = "nessie.server")
```

Comment:
switching this one currently still fails and I need to investigate why

Reply:
Did we have to upgrade Quarkus to fix this?

Reply:
yep, it was wrongly marked in their release notes that it's fixed in 2.1.3.Final, but it was actually fixed in 2.2.0.CR1